tests/src/FunctionalJavascript/ContributionIatsTest.php

```diff
@@ -82,22 +80,16 @@ public function testSubmit1stPayContribution() {
     ]));
     $this->enableCivicrmOnWebform();
 
-    $this->getSession()->getPage()->clickLink('Contribution');
-    $this->getSession()->getPage()->selectFieldOption('civicrm_1_contribution_1_contribution_enable_contribution', 1);
-    $this->assertSession()->assertWaitOnAjaxRequest();
-    $this->assertSession()->pageTextContains('You must enable an email field for Contact 1 in order to process transactions.');
-    $this->getSession()->getPage()->pressButton('Enable It');
-    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->configureContributionTab(FALSE, $this->payment_processor_faps['id']);
     $this->getSession()->getPage()->checkField('Contribution Amount');
-    $this->getSession()->getPage()->selectFieldOption('Currency', 'USD');
-    $this->getSession()->getPage()->selectFieldOption('Financial Type', 'Donation');
-    // throw new \Exception(var_export($this->getOptions('Payment Processor'), TRUE));
-    $this->assertCount(4, $this->getOptions('Payment Processor'));
-    $this->getSession()->getPage()->selectFieldOption('Payment Processor',  $this->payment_processor_faps['id']);
     $this->enableBillingSection();
 
     $this->saveCiviCRMSettings();
 
+    $this->submitWebformAndVerifyPayment();
+  }
```

Comment:
I think this needs `$isRecur` = 1 (or something)

Reply:
Hmm, this param was added for `testRecurringPayment()` which was removed in the second commit added to this PR? https://github.com/colemanw/webform_civicrm/pull/613/commits/e21e3edb17c9f3020984e603e89f6b56ba8a6667. The main commit has 2 tests calling this function. The above call is from `testSubmit1stPayContribution()` which is meant to do non-recur payment as it did before this PR. @KarinG